### PR TITLE
Edwards branch

### PR DESCRIPTION
**Pull request by Ejhuang2015**
Updated User model and added the Exhibit model. Added the Exhibit API route and updated the homeRoutes to include upload page, homepage, and index page.

### CODE REVIEW

File: controllers/api/exhibit.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// Dependencies
+// =============================================================
```
Comment (TheTiiiim) [inline controllers/api/exhibit.js:2]: these only appear in 2 files from your pull request and do not appear anywhere else in the repository

for consistency's sake these should be removed (or added everywhere)

File: views/fullstack.handlebars [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        </div>
+        <div class="card-content">
+            <p class="content">
+                {{???}}{{!-- Image of the artwork --}}
+            </p>
+        </div>
+    </div>
+    <hr>
+    {{/each}}
+</div>
```
Comment (TheTiiiim) [inline views/fullstack.handlebars:19]: moving away from short stacks and full stacks
but maybe we can use this in a gallery system

**Review by TheTiiiim — COMMENTED** [review]
I've left a few notes about style but otherwise it looks good

**Review by Ejhuang2015 — COMMENTED** [review]
Removed unnecessary parts

**Pull request merged**